pom.xml

```diff
@@ -67,7 +67,7 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.13</version>
-      <scope>test</scope>
+      <!-- <scope>test</scope> -->
```

Comment:
for some reason my tests we're working when I had that line. It told me the junit package wasn't found, so I googled for a bit and someone said to try it commented out and it worked (I didn't delete it because I'm not sure if this was a personal issue and wanted to leave it there in case tests didnt work for someone else)

Reply:
I added a comment just in case to clarify things!